docs/configuration.rst

```diff
@@ -412,6 +412,11 @@ The following environment variables for the tracer are supported:
      type: Boolean
      default: True
      description: Send query strings in http.url tag in http server integrations.
+    
+   DD_TRACE_SPAN_AGGREGATOR_RLOCK:
```

Comment:
This might be bike shedding, but I wonder if this should be a private env var?

`_DD_TRACE_SPAN_AGGREGATOR_RLOCK` since the `SpanAggregator` is a private implementation detail.

Reply:
It seems weird to suggest using a "private" envar to users though doesn't it? Up to you though!

Reply:
yeah, not sure personally. I am just thinking if we end up making this the default, easier to phase out if we change the default to RLock.

but this is also an env var so.... .... risks are low if someone is setting and we aren't doing something with it. not sure. would love input.